src/components/TransactionItem.tsx

```diff
@@ -21,11 +21,20 @@ const renderCofirmations = (txType: TxType, confirmations: number) =>
     <Text style={styles.label}>{`${i18n.transactions.list.conf}: ${getConfirmationsText(txType, confirmations)}`}</Text>
   );
 
-export const TransactionItem = ({ item, onPress }: { item: Transaction; onPress: (item: any) => void }) => {
+export const TransactionItem = ({
+  item,
+  onPress,
+  testID,
+}: {
+  item: Transaction;
+  onPress: (item: any) => void;
+  testID?: string;
+}) => {
```

Comment:
I think it'd be a good idea to create an interface named, for instance, `Props` to short this line a little

```
interface Props {
   item: Transaction;
   onPress: (item: any) => void;
   testID?: string;
}

export const TransactionItem = ({ item, onPress, testID }): Props => {}
```

Reply:
Done